blocklib/picoscope/test/qa_TimingMatcher.cc

```diff
@@ -535,6 +535,84 @@ const boost::ut::suite<"TimingMatchers"> TimingMatcherTests = [] {
             },
             result3.tags);
     };
+
+    "extended matching"_test = [&] {
```

Comment:
Should have a more meaningful name to say what this test is achtually checking.

I'm also not sure how i feel about having tests explicitly testing implementation details like `_lastMatchedTag` as this should not matter and shouldn't be exposed to the user.